RDSC-3619: Remapping the output (#1662)
redis/docs: content/integrate/redis-data-integration/data-pipelines/transform-examples/remapping-the-output.md

---
Title: Remapping the fields in the output
aliases: null
alwaysopen: false
categories:
- docs
- integrate
- rs
- rdi
description: null
group: di
linkTitle: Remapping the fields in the output
summary: Redis Data Integration keeps Redis in sync with the primary database in near
  real time.
type: integration
weight: 40
---

> Sometimes, you may want to remap the fields in the output of a data pipeline. This can be done by defining a `mapping` section in the output configuration.

Reviewer: ```suggestion
Sometimes, you may want to remap the fields in the output of a data pipeline. You can do this by defining a `mapping` section in the output configuration.
```
